WD-Firmware/psp/src/readme.txt

```diff
@@ -0,0 +1,23 @@
+/*

```

Comment:
dont take this readme it is not a license, its header. align the correct readme files for FreeRTOS to our files